tools/publish.sh

```diff
@@ -19,17 +19,16 @@
 set -eo pipefail
 
 # Install desired version of Node.js
-retry curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash >/dev/null
+retry curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash >/dev/null
 export NVM_DIR="$HOME/.nvm" >/dev/null
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" >/dev/null
 
-retry nvm install 10 &>/dev/null
+retry nvm install 20 &>/dev/null
```

Comment:
Is the number `20` reference to file descriptors ?
If so, why change `stdout(1)` to `stderr(2)`

Reply:
No, this is the version of NodeJS to install https://github.com/nvm-sh/nvm?tab=readme-ov-file#usage